### PR TITLE
Bug fix(Make sure the first user is admin) and docs update(Git commit message)

### PR DESCRIPTION
**Pull request by RileyYe**
I fixed the bug of creating admin account and formatted the code. 
Also, this PR includes the docs update.

### CODE REVIEW

File: back/src/graphql/type/user.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import { conn } from '../../db/connection.js';
 import {
+  GraphQLBoolean,
```
Comment (RileyYe) [inline back/src/graphql/type/user.js:3]: I don't know why their order changed, may be the result of `yarn format`...

Comment (PeterlitsZo) [reply back/src/graphql/type/user.js:3]: It's OK!

**Comment by PeterlitsZo** [comment]
Could you let the 1st commit and 3rd commit to be one commit?

**Comment by RileyYe** [comment]
Err, hard to, it seems I rebased a local branch and deleted it  :(

**Comment by PeterlitsZo** [comment]
> Because it will break 80-per-line rule.

But you break URL...

https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53?permalink_comment_id=2355038#gistcomment-2355038

I do not want to split URL into lines. It cannot be clickable.

**Comment by RileyYe** [comment]
No worries, there will be no indents for the left url parts, such that we can still copy them directly.





------------------ Original ------------------
From: peterlits zo ***@***.***&gt;
Date: Thu,Mar 17,2022 3:53 PM
To: ACMHomepage/ACMHomepage ***@***.***&gt;
Cc: RileyYe ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [ACMHomepage/ACMHomepage] fix(back)/docs: Bug fix and docs update(PR #20)





  
Because it will break 80-per-line rule.
  
But you break URL...
 
https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53?permalink_comment_id=2355038#gistcomment-2355038
 
I do not want to split URL into lines. It cannot be clickable.

**Comment by PeterlitsZo** [comment]
> No worries, there will be no indents for the left url parts, such that we can still copy them directly.

Do you have an example? In my opinion, I never find any open source project that will split URL into lines in the git commit message.

Thanks.

**Comment by RileyYe** [comment]
I just think it is reasonable to avoid the url displaying too long.

**Comment by PeterlitsZo** [comment]
But URL is not for display. Firstly it should be clickable. We can put it in a single paragraph or the last part. It will also be pretty and clickable.

**Comment by RileyYe** [comment]
I am not concerned. The decision is up to you...

**Review by PeterlitsZo — APPROVED** [review]
Done. We will add tests at another PR. :)

**Pull request merged**